packages/server/prod-server/src/server/modernServer.ts

```diff
@@ -353,28 +354,28 @@ export class ModernServer implements ModernServerInterface {
     return handler;
   }
 
-  protected async prepareAPIHandler(extension: APIServerStartInput['config']) {
-    const { workDir, runner, conf } = this;
-    const { bff } = conf as ConfWithBFF;
-    const prefix = bff?.prefix || '/api';
-    const webOnly = await isWebOnly();
-    if (webOnly && process.env.NODE_ENV === 'development') {
-      return (req: NodeRequest, res: NodeResponse) => {
-        res.setHeader('Content-Type', 'text/plain');
-        res.end(JSON.stringify(''));
-      };
-    }
-    return runner.prepareApiServer(
-      {
-        pwd: workDir,
-        config: extension,
-        prefix: Array.isArray(prefix) ? prefix[0] : prefix,
-        httpMethodDecider: bff?.httpMethodDecider,
-        render: this.render.bind(this),
-      },
-      { onLast: () => null as any },
-    );
-  }
+  // protected async prepareAPIHandler(extension: APIServerStartInput['config']) {
```

Comment:
Is unnecessary codes?

Reply:
Yeah, this's unnecessary code. Once we finish refactoring, we can remove the old prod-server as a whole